cmd/checkpoint/main.go

```diff
@@ -295,6 +351,11 @@ func writeCheckpointManifest(pod *v1.Pod) error {
 	return writeAndAtomicRename(path, b, 0644)
 }
 
+// isPodCheckpointer returns true if the pod is the checkpointer itself.
+func isPodCheckpointer(pod *v1.Pod) bool {
+	return strings.HasPrefix(pod.Name, "pod-checkpointer")
```

Comment:
Hmm actually, it won't have the node-name suffix in the checkpoint manifest - so maybe just `pod.Name == self.Name` if we care about the parent pod

or, if we care about the checkpointed copy, something like:

```
pod.Name == strings.TrimSuffix(self.Name, self.NodeName) && pod.Name != self.Name
```